pkg/controller/networkpolicy/adminnetworkpolicy.go

```diff
@@ -0,0 +1,405 @@
+}
+
+// updateAdminNP receives AdminNetworkPolicy UPDATE events and enqueues a
+// reference of the AdminNetworkPolicy to trigger its process.
+func (n *NetworkPolicyController) updateAdminNP(_, cur interface{}) {
+	defer n.heartbeat("updateAdminNP")
+	curANP := cur.(*v1alpha1.AdminNetworkPolicy)
+	klog.InfoS("Processing AdminNetworkPolicy UPDATE event", "anp", curANP.Name)
+	n.enqueueInternalNetworkPolicy(getAdminNPReference(curANP))
+}
+
+// deleteAdminNP receives AdminNetworkPolicy DELETE events and enqueues a
+// reference of the AdminNetworkPolicy to trigger its process.
+func (n *NetworkPolicyController) deleteAdminNP(old interface{}) {
+	anp, ok := old.(*v1alpha1.AdminNetworkPolicy)
+	if !ok {
+		tombstone, ok := old.(cache.DeletedFinalStateUnknown)
+		if !ok {
+			klog.Errorf("Error decoding object when deleting AdminNetworkPolicy, invalid type: %v", old)
+			return
+		}
+		anp, ok = tombstone.Obj.(*v1alpha1.AdminNetworkPolicy)
+		if !ok {
+			klog.Errorf("Error decoding object tombstone when deleting AdminNetworkPolicy, invalid type: %v", tombstone.Obj)
+			return
+		}
+	}
+	defer n.heartbeat("deleteAdminNP")
+	klog.InfoS("Processing AdminNetworkPolicy DELETE event", "anp", anp.Name)
+	n.enqueueInternalNetworkPolicy(getAdminNPReference(anp))
+}
+
+// addBANP receives BaselineAdminNetworkPolicy ADD events and enqueues a reference of
+// the BaselineAdminNetworkPolicy to trigger its process.
+func (n *NetworkPolicyController) addBANP(obj interface{}) {
+	defer n.heartbeat("addBANP")
+	banp := obj.(*v1alpha1.BaselineAdminNetworkPolicy)
+	klog.InfoS("Processing BaselineAdminNetworkPolicy ADD event", "banp", banp.Name)
+	n.enqueueInternalNetworkPolicy(getBANPReference(banp))
+}
+
+// updateBANP receives BaselineAdminNetworkPolicy UPDATE events and enqueues a
+// reference of the BaselineAdminNetworkPolicy to trigger its process.
+func (n *NetworkPolicyController) updateBANP(_, cur interface{}) {
+	defer n.heartbeat("updateBANP")
+	curBANP := cur.(*v1alpha1.BaselineAdminNetworkPolicy)
+	klog.InfoS("Processing AdminNetworkPolicy UPDATE event", "banp", curBANP.Name)
+	n.enqueueInternalNetworkPolicy(getBANPReference(curBANP))
+}
+
+// deleteBANP receives BaselineAdminNetworkPolicy DELETE events and enqueues a
+// reference of the BaselineAdminNetworkPolicy to trigger its process.
+func (n *NetworkPolicyController) deleteBANP(old interface{}) {
+	banp, ok := old.(*v1alpha1.BaselineAdminNetworkPolicy)
+	if !ok {
+		tombstone, ok := old.(cache.DeletedFinalStateUnknown)
+		if !ok {
+			klog.Errorf("Error decoding object when deleting BaselineAdminNetworkPolicy, invalid type: %v", old)
+			return
+		}
+		banp, ok = tombstone.Obj.(*v1alpha1.BaselineAdminNetworkPolicy)
+		if !ok {
+			klog.Errorf("Error decoding object tombstone when deleting BaselineAdminNetworkPolicy, invalid type: %v", tombstone.Obj)
+			return
+		}
+	}
+	defer n.heartbeat("deleteBANP")
+	klog.InfoS("Processing BaselineAdminNetworkPolicy DELETE event", "banp", banp.Name)
```

Comment:
You may adjust the log level to `klog.V(2).InfoS()` in the different events handle methods.

Reply:
Fixed